package.json

```diff
@@ -73,6 +72,7 @@
     "lint-staged": "8.1.0",
     "mocha": "5.2.0",
     "mocha-headless-chrome": "2.0.1",
+    "npm-run-all": "^4.1.3",
```

Comment:
the ^ is not needed!

Reply:
no problem, will fix! Out of curiosity, why do you pin the version numbers? Especially the production version numbers, that can cause that multiple versions of the same dependency are necessary, as the pinned version prevents deduping of packages

Reply:
Been explained here: #13